src/json2object/utils/schema/DataBuilder.hx

```diff
@@ -42,12 +42,46 @@ typedef Definitions = Map<String, JsonType>;
 
 class DataBuilder {
 
-	static var BOOL = Context.getType("Bool");
-	static var FLOAT = Context.getType("Float");
-	static var STRING = Context.getType("String");
+	@:persistent
+	static var cachedBool:Type = null;
```

Comment:
You don't want to have `Type`s as `@:persistent`

Reply:
Thanks. That was one thing I was unsure of, just forgot to mention it in the main post.